pkg/liqoctl/completion/completion.go

```diff
@@ -143,6 +146,54 @@ func VirtualNodes(ctx context.Context, f *factory.Factory, argsLimit int) FnType
 	return common(ctx, f, argsLimit, retriever)
 }
 
+// LabelsSelector returns a function to autocomplete selector labels.
+func LabelsSelector(ctx context.Context, f *factory.Factory, argsLimit int) FnType {
+	retriever := func(ctx context.Context, f *factory.Factory) ([]string, error) {
+		tenantnamespaces := corev1.NamespaceList{}
+		if err := f.CRClient.List(ctx, &tenantnamespaces, client.MatchingLabels{
+			discovery.TenantNamespaceLabel: "true",
+		}); err != nil {
+			return nil, err
+		}
+		labelset := map[string]int{}
+		var virtualNodes virtualkubeletv1alpha1.VirtualNodeList
+		if err := f.CRClient.List(ctx, &virtualNodes); err != nil {
+			return nil, err
+		}
+		for i := range virtualNodes.Items {
+			l, err := utilsvirtualnode.GetLabelSelectors(ctx, f.CRClient, &virtualNodes.Items[i])
```

Comment:
```suggestion
			labelSet, err := utilsvirtualnode.GetLabelSelectors(ctx, f.CRClient, &virtualNodes.Items[i])
```

`l` does mean nothing, please use better names to improve the readability of the code